DISCOVER/06_childcare.md

```diff
@@ -24,7 +24,7 @@
  
 - There are companies that provide childcare at conferences. While NumFOCUS makes no specific endorsements, we did find these companies in our research who are providing event-based childcare:
    - [Corporate Kids Events](https://conferencechildcare.com/)
-   - [American Child Care, Inc.](http://www.americanchildcare.com/conventions.htm)
+   - [American Child Care, Inc.](https://www.americanchildinc.org/)
```

Comment:
Please remove the link rather than use this one which is not about providing event-based childcare.